Dockerfile

```diff
@@ -16,12 +16,12 @@ FROM debian:11-slim@sha256:f7d141c1ec6af549958a7a2543365a7829c2cdc4476308ec2e182
 
 RUN apt update && apt install -y curl sqlite3 luajit libboost-dev libboost-program-options-dev && apt clean
 
-# REMINDER: .dockerignore defaults to exclude everything. Add exceptions to be copied there.
-ADD entrypoint.sh /entrypoint/script
-
 COPY --from=builder /usr/local /usr/local
 
+# REMINDER: .dockerignore defaults to exclude everything. Add exceptions to be copied there.
+ADD zone-populator.sh /usr/local/bin/zone-populator.sh
+
```

Comment:
As caches work fantastically good and the images are pinned to the hash, it is more likely to change the script than the layers below.

This should make the build even faster.